scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -196,12 +195,6 @@ def main():
         raise Exception('The e2e test suites that have been extracted from '
                         'protractor.conf.js are empty.')
 
-    if SAMPLE_TEST_SUITE_THAT_IS_KNOWN_TO_EXIST not in travis_e2e_jobs:
-        raise Exception('{} is expected to be in the e2e test suites '
```

Comment:
Why removing this?

Reply:
The test was assuming that the `RUN_E2E_TESTS_...`  part was correlated with the suite name. Because I'm grouping tests, that assumption doesn't make sense any more.

Reply:
I see, in that case should we use `travis_e2e_suites` instead of `travis_e2e_jobs`? will that fix the issue?

Reply:
yup, changed!